src/pages/HomePage/MovieCard/styles.js

```diff
@@ -0,0 +1,57 @@
+import styled from 'styled-components';
+import { Link } from 'react-router-dom';
+
+export const Description = styled.div`
+	background-color: rgba(53, 63, 76, 0.9);
+	display: flex;
+	cursor: pointer;
+	flex-direction: column;
+	height: 100%;
+	margin-top: calc((100vw - 2rem) * 1.333);
+	padding: 0.5rem;
+	transition: 0.2s ease;
```

Comment:
Prefer to define which rule you would like to do the transition. As you're making the transition of every property, you can face unexpected behaviours, but if you're sure of it, be more explicit and put `all` instead of nothing. 

```css
transition: all 200ms ease;
```